pkgs/data/fonts/tamzen/default.nix

```
@@ -0,0 +1,49 @@
+{ fetchFromGitHub, fontforge, mkfontscale, stdenv }:
+
+let
+  version = "1.11.4";
+in stdenv.mkDerivation {
+  pname = "tamzen-font";
+  inherit version;
```

use recursive attr set
```suggestion
stdenv.mkDerivation rec {
  pname = "tamzen-font";
  version = "1.11.4";
```